PRESTO_ELASTICSEARCH_CONNECTOR_TUTORIAL

```diff
@@ -0,0 +1,132 @@
+
+To check whether ElasticSearch has correctly installed and started locally, use the following URL in browser :
+
+  http://localhost:9200/
+
+
+
+It should show you an output like:
+
+
+{
+  "name" : "ip-172-31-18-24.ap-south-1.compute.internal",
+  "cluster_name" : "elasticsearch",
+  "cluster_uuid" : "wY9BlfS-TKqTaCRpMi8kDQ",
+  "version" : {
+    "number" : "8.8.1",
+    "build_flavor" : "default",
+    "build_type" : "tar",
+    "build_hash" : "f8edfccba429b6477927a7c1ce1bc6729521305e",
+    "build_date" : "2023-06-05T21:32:25.188464208Z",
+    "build_snapshot" : false,
+    "lucene_version" : "9.6.0",
+    "minimum_wire_compatibility_version" : "7.17.0",
+    "minimum_index_compatibility_version" : "7.0.0"
+  },
+  "tagline" : "You Know, for Search"
+}
+
+
+
+
+
+
+## Adding data to index
+
+curl -XPOST -H "Content-Type: application/json" localhost:9200/samples/_bulk -d '
+{"index": {"_id": "975463711"}}
+{"Amount": "480", "Quantity": "2", "Id": "975463711", "Client_Store_sk": "1109"}
+{"index": {"_id": "975463943"}}
+{"Amount": "2105", "Quantity": "2", "Id": "975463943", "Client_Store_sk": "1109"}
+'
+
+
+[ec2-user@ip-172-31-18-24 elasticsearch-8.8.1]$ curl -XPOST -H "Content-Type: application/json" localhost:9200/samples/_bulk -d '
+{"index": {"_id": "975463711"}}
+{"Amount": "480", "Quantity": "2", "Id": "975463711", "Client_Store_sk": "1109"}
+{"index": {"_id": "975463943"}}
+{"Amount": "2105", "Quantity": "2", "Id": "975463943", "Client_Store_sk": "1109"}
+'
+{"took":87,"errors":false,"items":[{"index":{"_index":"samples","_id":"975463711","_version":1,"result":"created","_shards":{"total":2,"successful":1,"failed":0},"_seq_no":3,"_primary_term":4,"status":201}},{"index":{"_index":"samples","_id":"975463943","_version":1,"result":"created","_shards":{"total":2,"successful":1,"failed":0},"_seq_no":4,"_primary_term":4,"status":201}}]}
+
+
+
+## Deploying Presto
+
+Deploy presto using below Documentation
+
+https://prestodb.io/docs/current/installation/deployment.html
+
+Elasticsearch Connector Configuration#
+
+To configure the Elasticsearch connector, create a catalog properties file etc/catalog/elasticsearch.properties with the following contents, replacing the properties as appropriate:
+
+connector.name=elasticsearch
+elasticsearch.host=localhost
+elasticsearch.port=9200
+elasticsearch.default-schema-name=default
+
+
+
+## Presto-Command Line Interface
+
+The Presto CLI provides a terminal-based interactive shell for running queries. The CLI is a self-executing JAR file, which means it acts like a normal UNIX executable.
+
+Download presto-cli-0.281-executable.jar, rename it to presto, make it executable with chmod +x, then run it:
+
+./presto --server localhost:8080 --catalog elasticsearch --schema default
+Run the CLI with the --help option to see the available options.
+
+$ ./presto --server localhost:8080 --catalog elasticsearch --schema default
+
+presto> show tables in elasticsearch.default;
```

Comment:
1. Explain what the user is doing. "At the Presto prompt, run the following command: 
2. Suggest formatting the command on the next line as code